--- mds-bridge.js	2021-03-31 15:12:17.000000000 -0700
+++ /home/jas/mds-bridge.js	2021-04-03 07:54:29.838341655 -0700
@@ -35,7 +35,7 @@
     var configJson = fs.readFileSync('mds-bridge.json');
     var configObj = JSON.parse(configJson.toString());
     console.log(configObj);
-    debug = configObj.debug;
+        if ( debug >= 1)debug = configObj.debug;
     webSocketPort = configObj.port;
     refreshRate = configObj.refresh;
 }catch(error){} //keep defaults
@@ -527,7 +527,9 @@
 function handleEventJSONReception(evMessage)
 {
     var configObj = JSON.parse(evMessage);
-    console.log(configObj);
+    if ( debug >= 1) {
+	console.log(configObj);
+    }
     var shot = configObj.shot;
     var name = configObj.name;
     var samples = configObj.samples;
