diff --git a/GAMs/PyGAM/Makefile.inc b/GAMs/PyGAM/Makefile.inc
index 7b6647c..36f1af5 100644
--- a/GAMs/PyGAM/Makefile.inc
+++ b/GAMs/PyGAM/Makefile.inc
@@ -49,7 +49,7 @@ CPPFLAGS += -Wno-unused-function
 ####THE FOLLOWING DEFINITIONS MUST BE CHANGED IN ORDER TO ADAPT 
 ####TO THE INSTALLED PYTHON (>=3.7)
 # Python header file folder
-INCLUDES += -I/usr/include/python3.6m/
+INCLUDES += -I/usr/include/python3.10/
 INCLUDES += -I/usr/lib/python3/dist-packages/numpy/core/include/
 # Python interpreter folder
 PYINTERPRETER = /usr/lib/arm-linux-gnueabihf/
@@ -59,7 +59,7 @@ PYINTERPRETER = /usr/lib/arm-linux-gnueabihf/
 
 
 # Linker flags requested by python3.7-config
-LIBRARIES += -L$(PYINTERPRETER) -lpython3.6m -lutil -Xlinker -export-dynamic
+LIBRARIES += -L$(PYINTERPRETER) -lpython3.10 -lutil -Xlinker -export-dynamic
 
 all: $(OBJS) $(SUBPROJ) \
 	$(BUILD_DIR)/PyGAM$(DLLEXT)
